cdap-docs/_common/common-build.sh

```diff
@@ -108,7 +108,7 @@ GITHUB="github"
 
 # BUILD.rst
 BUILD_RST="BUILD.rst"
-BUILD_RST_HASH="f54ae74bb72f9ad894766b6c0bd2d2df"
+BUILD_RST_HASH="79a46db1e9fb3d428c384bdea0c3d596"
```

Comment:
So when the BUILD.rst changed, I notice it, as the build.sh scripts use the same values.